src/qiboconnection/api.py

```diff
@@ -415,6 +445,9 @@ def get_result(self, job_id: int) -> CircuitResult | npt.NDArray | dict | None:
             CircuitResult | npt.NDArray | dict | None: The Job result as an Abstract State or None when it is not
             executed yet.
         """
+        warnings.warn(
+            "This method is deprecated and will be removed in a future qiboconnection version. Use get_job(job_id).result to retrieve your results instead."
```

Comment:
"in a future qiboconnection version".

How about hardcoding that version?

We could say that the next major version (qiboconnection v1.0.0?) will have backwards-incompatible changes, does it make sense?

(No need to change it right now, just to start thinking on a deprecation policy)

Reply:
makes sense to me, although I think we may be quite far from releasing qiboconnection 1.0